namui-animation-editor/src/graph_editor/graph_window/render/render_graph/f32_based.rs

```diff
@@ -0,0 +1,330 @@
+use std::fmt::Display;
+
+use super::*;
+use namui::animation::KeyframePoint;
```

Comment:
```suggestion
use super::*;
use namui::animation::KeyframePoint;
use std::fmt::Display;
```

This is a result of linting imports